soc/st/stm32/stm32l0x/soc.c

```diff
@@ -33,12 +33,7 @@ static int stm32l0_init(void)
 	/* At reset, system core clock is set to 2.1 MHz from MSI */
 	SystemCoreClock = 2097152;
 
-	/* Default Voltage scaling range selection (range2)
-	 * doesn't allow to configure Max frequency
-	 * switch to range1 to match any frequency
-	 */
 	LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
```

Comment:
Let's move PWR clock activation in `config_enable_default_clocks` function then.
Related include could be cleaned up